libfreerdp/core/redirection.c

```diff
@@ -373,14 +357,23 @@ static state_run_t rdp_recv_server_redirection_pdu(rdpRdp* rdp, wStream* s)
 		return STATE_RUN_FAILED;
 
 	Stream_Read_UINT16(s, flags);                  /* flags (2 bytes) */
+	if (flags != SEC_REDIRECTION_PKT)
+	{
+		char buffer1[1024] = { 0 };
+		char buffer2[1024] = { 0 };
+		WLog_ERR(TAG, "[%s] received invalid flags=%s, expected %s", __FUNCTION__,
+		         rdp_security_flag_string(flags, buffer1, sizeof(buffer1)),
+		         rdp_security_flag_string(SEC_REDIRECTION_PKT, buffer2, sizeof(buffer2)));
```

Comment:
Same as above - why not just use "SEC_REDIRECTION_PKT"

Reply:
`rdp_security_flag_string` does some nice formatting (e.g. stringify and add the number in brackets)